app/views/timed_service_price_policies/_adjustment_row.html.haml

```diff
@@ -0,0 +1 @@
+= render "time_based_price_policies/adjustment_row", price_group: price_group, price_policy: price_policy, cancellation: false, minimum_cost: false
```

Comment:
Waggles and I discussed, and we concluded that having the `cancellation` and `minimum_cost` locals explicitly assigned here helps to make it more clear what the purpose of these partials are.